wgengine/magicsock/endpoint.go

```diff
@@ -866,15 +879,36 @@ func (de *endpoint) addCandidateEndpoint(ep netip.AddrPort, forRxPingTxID stun.T
 	return false
 }
 
+// clearBestAddrLocked clears the bestAddr and related fields such that future
+// packets will re-evaluate the best address to send to next.
```

Comment:
```suggestion
// clearBestAddrLocked clears the bestAddr and related fields such that future
// packets will re-evaluate the best address to send to next.
//
// b.mu must be held.
```
I believe we follow the convention that we add this type of comment for functions like this.